tests/test_disclosure_control_check.py

```diff
@@ -1,28 +1,36 @@
 """disclosure_control_check tests"""
+import sys
+
+sys.path.append("../src")
+
 import unittest
 import doctest
 import pandas as pd
-from src.disclosure_control_check import check_series_sdc
-import src.make_data
-
-
-def load_tests(loader, tests, ignore):  # pylint: disable=unused-argument
-    """This creates a unittest.TestCase from the doctests described in the
-    module
-    """
-    tests.addTests(doctest.DocTestSuite(src.disclosure_control_check))
-    return tests
+#from src.disclosure_control_check import check_series_sdc
+import src.disclosure_control_check as src_disc
+# def load_tests(loader, tests, ignore):  # pylint: disable=unused-argument
+#     """This creates a unittest.TestCase from the doctests described in the
+#     module
+#     """
+#     tests.addTests(doctest.DocTestSuite(src.disclosure_control_check))
+#     return tests
 
 
 class TestCheckSeriesSdc(unittest.TestCase):
     """Testing the check_series_sdc function"""
 
-    def output_correct(self):
+    def test_output_correct(self):
         """checks the output of the function is as expected"""
-        data = pd.Series([10, 15, 20, 25, 30])
-        result = check_series_sdc(data).tolist()
-        expected = pd.Series([True, True, True, True, True]).tolist()
-        self.assertEqual(result, expected, "incorrect check for sdc")
+        self.assertEqual(src_disc.check_series_sdc(pd.Series([10, 15, 20, 25, 30])).tolist(), 
+                        pd.Series([True, True, True, True, True]).tolist(), 
+                        "incorrect check for sdc")
+        self.assertEqual(src_disc.check_series_sdc(pd.Series([5, 31, 1005, -5, -100])).tolist(), 
+                        pd.Series([False, False, True, False, True]).tolist(), 
+                        "incorrect check for sdc")
+        self.assertEqual(src_disc.check_series_sdc(pd.Series(["Egg", 27, -17, "Adam", 10052022])).tolist(), 
+                        pd.Series([False, False, False, False, False]).tolist(),  # should be false as the value is date like
```

Comment:
not sure I agree with this one - we can't stop every "date-like" value - might be worth discussing this, as it's not currently implemented in the SDC function.

This sounds like a different "check", e.g. confirming dates aren't into numeric fields, or whatever,